fusion-cli/build/modern-browser-versions.js

```diff
@@ -11,4 +11,5 @@ module.exports = {
   chrome: 61,
   safari: 12,
   firefox: 60,
+  edge: 79,
```

Comment:
Is the babel env preset aware of edge chromium?

Reply:
Yes, babel env preset uses caniuse-lite which is aware of chromium edge.